addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -694,4 +698,5 @@
     <i t-if="slide.slide_type == 'quiz'" t-att-class="'fa fa-question-circle %s' % icon_class"></i>
 </template>
 
+
```

Comment:
This diff shouldn't be in this particular commit.

([IMP] website_slides: improve the ui on a course (documentation))

Reply:
That comment did not get fix, even though it's not a big deal.

Reply:
Is that comment fixed :D ?